fdbrpc/include/fdbrpc/LoadBalance.actor.h

```diff
@@ -745,13 +772,17 @@ Future<REPLY_TYPE(Request)> loadBalance(
 				when(ErrorOr<REPLY_TYPE(Request)> result =
 				         wait(firstRequestData.response.isValid() ? firstRequestData.response : Never())) {
 					if (firstRequestData.checkAndProcessResult(atMostOnce)) {
+						//wait(firstRequestData.maybeDoReplicaComparison(
```

Comment:
Enabling this and the call below is causing a failure. It is probably because I am not using "resetReply()" properly in "maybeDoReplicaComparison()"?

Reply:
Addressed now.